### PR TITLE
bins: add cloud-hypervisor extra package

### PR DESCRIPTION
**Pull request by maxux**
Needed for #1061 
https://hub.grid.tf/tf-autobuilder/cloud-hypervisor-0.11.0.flist.md

### CODE REVIEW

File: bins/packages/cloudhypervisor/cloudhypervisor.sh [hunk 1]
```diff
@@ -0,0 +1,62 @@
+CLOUDHYPERVISOR_VERSION="0.13.0"
+CLOUDHYPERVISOR_CHECKSUM="44bd96c255b8d573a83eef2364d934ae"
+CLOUDHYPERVISOR_LINK="https://github.com/cloud-hypervisor/cloud-hypervisor/archive/v${CLOUDHYPERVISOR_VERSION}.tar.gz"
+CLOUDHYPERVISOR_RUST_TOOLCHAIN="1.48.0"
```
Comment (LeeSmet) [inline bins/packages/cloudhypervisor/cloudhypervisor.sh]: Is there any reason we are building with this version and not latest stable (currently 1.50)?

File: pkg/vm/ch.go [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+	tmp, err := ioutil.TempFile("", "vm-exec-*.sh")
+	if err != nil {
+		return errors.Wrap(err, "failed to create vm exec script")
+	}
+
+	log.Debug().Str("name", m.ID).Str("exec", tmp.Name()).Msg("using exec file")
+	defer func() {
+		tmp.Close()
+		//os.Remove(tmp.Name())
```
Comment (LeeSmet) [inline pkg/vm/ch.go]: commented code

**Review by LeeSmet — CHANGES_REQUESTED** [review]

File: bins/packages/cloudhypervisor/cloudhypervisor.sh (hunk 1 above)
Comment (muhamadazmy) [reply bins/packages/cloudhypervisor/cloudhypervisor.sh]: no

File: pkg/vm/ch.go (hunk 2 above)
Comment (muhamadazmy) [reply pkg/vm/ch.go]: ah, good catch

**Review by LeeSmet — APPROVED** [review]
:ship:

**Pull request merged**